DeBERTa/deberta/disentangled_attention.py

```diff
@@ -174,7 +174,9 @@ def linear(w,b,x):
         if self.talking_head:
             attention_scores = self.head_logits_proj(attention_scores.permute(0,2,3,1)).permute(0,3,1,2)
 
-        attention_probs = XSoftmax.apply(attention_scores, attention_mask, -1)
+        #attention_probs = XSoftmax.apply(attention_scores, attention_mask, -1)
+        nodex = torch.nn.Softmax(-1)
+        attention_probs = nodex(attention_scores)
```

Comment:
You need to apply the mask to the attention_scores by add it with a -inf number, e.g. -10000

Reply:
Something like this ? : 
attention_probs = nodex(attention_scores * attention_mask)

Reply:
And what about Dropout changes? How to apply mask there?